styles.css

```diff
@@ -80,15 +93,23 @@ body {
   padding: 10px;
   font-size: 16px;
   border-radius: 10px;
+  margin-top: 20px;
```

Comment:
Same as in exercise 3. If you want to build a responsive app, in this case it would be better to use `flex-grow` instead of `margin`